extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -170,18 +178,52 @@ require([
 		event.preventDefault();
 	});
 
-	$(document).on( 'click', '#modalTabAdmins', function () {
+	$(document).on( 'click', '#modalTabAdmins', function (event) {
 		switchCommunityModalTab(tabs.TAB_ADMINS);
 		event.preventDefault();
 	});
 
-	$(document).on( 'click', '#modalTabLeaderboard', function () {
+	$(document).on( 'click', '#modalTabLeaderboard', function (event) {
 		switchCommunityModalTab(tabs.TAB_LEADERBOARD);
 		event.preventDefault();
 	});
 
+	function initTracking() {
+		// Track clicks in contribution module
+		$('.ContributorsModule').on('mousedown touchstart', 'a',  function (event) {
+			var data = $(event.currentTarget).data('tracking');
+
+			if (typeof(data) !== 'undefined') {
+				track({
+					label: data,
+				});
+			}
+		});
+	}
+
+	function initModalTracking() {
+		// Track clicks in contribution modal
+		$('#CommunityPageModalDialog').on('mousedown touchstart', 'a', function (event) {
+			var data = $(event.currentTarget).data('tracking');
```

Comment:
This handler function is exactly the same as the one for the module tracking above, so I'd move it out into a `handleClick` method and then reuse it for both.